.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@mikekistler  Added this vscode settings file to avoid linting issues due to LF->CRLF issue. 
Since most of MS developers will be using Windows machine so it will be helpful for them to automatically fix line ending issues.